storey/aggregations.py

```diff
@@ -251,9 +251,17 @@ def __init__(self, features: List[str], table: Union[Table, str], key: Union[str
                  fixed_window_type: Optional[FixedWindowType] = FixedWindowType.CurrentOpenWindow, **kwargs):
         self._aggrs = []
         self._enrich_cols = []
+        self._table = None
         resolved_aggrs = {}
+        Flow.__init__(self, **kwargs)
```

Comment:
`AggregateByKey.__init__` already calls `Flow.__init__`. So this will cause it to be called twice.